nginx/conf.d/scalelite.template

```diff
@@ -4,5 +4,22 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
-	include conf.d/scalelite.common;
+	location / {
+		proxy_pass http://scalelite-api:3000;
+
+		proxy_read_timeout 60s;
+		proxy_redirect off;
+
+		proxy_set_header  Host $http_host;
+
+		proxy_set_header  X-Real-IP $remote_addr;
+		proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+		proxy_set_header  X-Forwarded-Proto $http_x_forwarded_proto;
```

Comment:
Do not unconditionally add the `proxy_set_header  X-Forwarded-Proto $http_x_forwarded_proto;` since it means the proto can be spoofed if scalelite ports are directly accessible. (Admittedly not a *big* deal here since scalelite doesn't use it for anything, but still.)

We should probably have a separate config for "behind an external https-terminating load balancer" which sets the X-Forwarded-Proto header and uses the [nginx realip module](https://nginx.org/en/docs/http/ngx_http_realip_module.html) to pull the original ip from the forwarded request to log and pass to scalelite.

Reply:
Is it not be safe to assume that anyone not using SSL would be placing Scalelite behind a network load balancer?